server/models/bhima.sql

```diff
@@ -98,7 +98,7 @@ INSERT INTO `report` (`id`, `report_key`, `title_key`) VALUES
   (6, 'account_report', 'REPORT.REPORT_ACCOUNTS.TITLE'),
   (7, 'cashflowByService', 'REPORT.CASHFLOW_BY_SERVICE'),
   (8, 'openDebtors', 'REPORT.OPEN_DEBTORS.TITLE'),
-  (9, 'clients_report','REPORT.CLIENTS_REPORT'),
+  (9, 'clients_report','REPORT.CLIENTS'),
```

Comment:
I think the translation key should be `REPORT.CLIENTS_REPORT`.  The i18n key `REPORT.CLIENTS` hasn't been defined.